trip-service/docs/trip-events.md

```
@@ -0,0 +1,58 @@
+# Trip Service Event Schemas
+
+Цей документ описує структуру повідомлень, які `TripService` використовує для взаємодії з іншими сервісами через Message Broker.
+
+### Версіонування
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)